src/main/java/org/myrobotlab/framework/Platform.java

```diff
@@ -393,11 +393,11 @@ public boolean isX86() {
         // zf.close(); explodes on closing :(
       } else {
         // IDE - version ...
-        in = new FileInputStream("target/classes/META-INF/MANIFEST.MF");// Platform.class.getResource("target/classes/META-INF/MANIFEST.MF").openStream();
+        // in = new FileInputStream("target/classes/META-INF/MANIFEST.MF");// Platform.class.getResource("target/classes/META-INF/MANIFEST.MF").openStream();
+        in = new FileInputStream("target/classes/git.properties");// Platform.class.getResource("target/classes/META-INF/MANIFEST.MF").openStream();

```

Comment:
Where does this git.properties come from?

Reply:
generated by the maven build